### ISSUE TITLE
nested kernel write protections

### ISSUE THREAD

**Issue by opntr** [HardenedBSD/hardenedBSD#80]

**Comment by opntr** [comment]
Please reopen this ticket... I hate github's auto closer function.

**Comment by opntr** [comment]
Could you please reopen this issue. It was accidentally closed by ZFS
related upstream commit.

**Comment by lattera** [comment]
Reopening issue.

**Comment by ndauten** [comment]
Can we reopen this issue? I haven't had time to build it myself, but I have had students working on porting off an on for the past year. I have not given up, just focusing where job demands are going.

**Comment by lattera** [comment]
Yup. It looks like the commit that was merged in by our auto-sync bot closed the ticket. We didn't close it on purpose.

**Comment by ndauten** [comment]
No worries. That's what I figured, and I'm sorry this hasn't moved faster. It will definitely provide a lot more security to the system. Also, if you have folks who are interested and capable of MMU dev, I'd be willing to mentor. Just to keep in mind.
